### PR TITLE
refactor(terraform): reorganize Cloud Build handling in ops project

### PR DESCRIPTION
**Pull request by grayside**
Part of #497.

This PR contains the following changes:

* Light reorganization of resources in the Ops module
* Rename of the `deploy_triggers` variable to `cd_system`
* Updated trigger resource definitions for all "push-to-main" build and testing triggers, and the nightly e2e testing trigger.
* Moving build triggers prompt inside the terraform section of setup.sh
* Whitespace adjustments around the terraform section of setup.sh
* (Temporary) Default disable of trigger setup in setup.sh

Follow-up changes will continue implementation of #497 and #498.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ trap '_error_report $LINENO' ERR
 
 # Default to empty, avoiding unbound variable errors.
 SKIP_TERRAFORM=${SKIP_TERRAFORM:-}
-SKIP_TRIGGERS=${SKIP_TRIGGERS:-}
+SKIP_TRIGGERS=${SKIP_TRIGGERS:-1} # TEMPORARY: Suppress triggers during iterative fix.
```
Comment (ace-n) [inline setup.sh:40]: I assume this was intentional (i.e. not a "debugging statement")?

File: setup.sh [hunk 2]
```diff
@@ -60,39 +60,74 @@ export REGION="us-central1"
...
+    terraform -chdir=${STAGE_ENVIRONMENT_DIR} init
+    terraform -chdir=${STAGE_ENVIRONMENT_DIR} apply --auto-approve
+
+    # Prod Project
+    # Only deploy to separate project for multi-project setups
+    if [ "${PROD_PROJECT}" != "${STAGE_PROJECT}" ]; then 
+        PROD_ENVIRONMENT_DIR=terraform/environments/prod
+        export TF_VAR_project_id=${PROD_PROJECT}
+        export TF_VAR_ops_project_id=${OPS_PROJECT}
+        export TF_VAR_deploy_trigger_project=${STAGE_PROJECT}
```
Comment (ace-n) [inline setup.sh:127]: I assume `STAGE_PROJECT` (vs. `PROD_PROJECT`) isn't a typo here. 🙂

File: terraform/environments/ops/variables.tf [hunk 3]
```diff
@@ -9,10 +9,10 @@ variable "region" {
   description = ""
 }
 
-variable "deploy_triggers" {
+variable "cd_system" {
```
Comment (ace-n) [inline terraform/environments/ops/variables.tf:12]: `deploy` feels like a verb here - can we call this `setup_cd_system` or something like that?

File: terraform/modules/ops/main.tf [hunk 4]
```diff
@@ -21,25 +23,9 @@ resource "google_project_iam_member" "pubsub_publisher_iam_member" {
   ]
 }
 
-# Container Hosting
-
-# Artifact Registry API enablement is eventually consistent
```
Comment (ace-n) [inline terraform/modules/ops/main.tf:26]: Is there a reason we're deleting these comments?

(Are they no longer relevant? If they are, I'd argue this information should be documented _somewhere_, e.g. an `.md` file.)

File: terraform/modules/ops/main.tf [hunk 5]
```diff
@@ -79,18 +65,12 @@ resource "google_artifact_registry_repository" "e2e_runner_docker" {
...
 
-###
-# Secret Manager
-###
+###########################
+# End-user Authentication #
+###########################
 
-# OAuth 2.0 secrets
-# These secret resources are REQUIRED, but configuring them is OPTIONAL.
```
Comment (ace-n) [inline terraform/modules/ops/main.tf:87]: Ditto - is there a reason we're deleting these comments?

**Review by ace-n — APPROVED** [review]
LGTM once comments addressed.

File: setup.sh (hunk 2 above)
Comment (grayside) [reply setup.sh:127]: No, it's not a typo. (Also, it will be gone in the next push.)

File: terraform/environments/ops/variables.tf (hunk 3 above)
Comment (grayside) [reply terraform/environments/ops/variables.tf]: fixed

File: setup.sh (hunk 2 above)
Comment (grayside) [reply setup.sh]: fixed

File: terraform/modules/ops/main.tf (hunk 4 above)
Comment (grayside) [reply terraform/modules/ops/main.tf:26]: These comments were moved to services.tf.

File: terraform/modules/ops/main.tf (hunk 5 above)
Comment (grayside) [reply terraform/modules/ops/main.tf:87]: I did some comment refactoring, but I think I lost something. Will fix.

**Pull request merged**